Add examples integrate Flux & Turing to Stheno (#84)
willtebbutt/Stheno.jl: examples/flux_integration/simple_fnn/README.md

# Neural network + Gaussian process ( step function )
## Introduction

> Gaussian process is typically used as a smoothing device, here, we show that by combining with a simple fully connected neural network, it's able to correctly learn sharp discontinuities.

Reviewer: ```suggestion
A Gaussian process is typically used as a smoothing device. Here we show that by combining with a simple fully connected neural network it's able to learn sharp discontinuities, as described in [2].
```

Targeted function:
$$
\begin{gather}
y=F(x)+\epsilon,\;\epsilon\sim\mathcal{N}(0, 0.01^2)\\
F(x)=\left\{\begin{array}{rl}
    0.0 & \text{if } x\le0.0\\
    1.0 & \text{if } x>0.0
    \end{array}\right.
\end{gather}
$$

> Neural network is used as a feature extractor, it can be viewed as a nonlinear transformation $T:\,x\to z$, Gaussian process with Automatic Relevance Determination (ARD) kernel is then used to map $z\to y$, the combined mapping can be represented as $y=f(T(x))$.

Reviewer: ```suggestion
The neural network is used as a feature extractor, it can be viewed as a nonlinear transformation $T:\,x\to z$, and a Gaussian process with an Automatic Relevance Determination (ARD) kernel is then used to map $z\to y$. The combined mapping can be represented as $y=f(T(x))$.
```

> The results is show below:

Reviewer: ```suggestion
The result is show below:
```
